lib/ansible/modules/cloud/amazon/ec2_elb_facts.py

```diff
@@ -80,6 +80,9 @@
 
 import traceback
 
+from ansible.module_utils.basic import *
```

Comment:
Ah, I see you've just moved the code to satisfy CI, rather than these been new imports. If you could fix this it would be great.